### PR TITLE
Tweak: Add plugins support to the CLI import process [ED-6902]

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: core/app/modules/import-export/wp-cli.php [hunk 1]
```diff
@@ -216,4 +225,62 @@ private function create_temp_file_from_url( $url ) {
...
+		}
+
+		$manifest_data = json_decode( $manifest_file_content, true );
+
+		// In case that the manifest content is not a valid JSON or empty.
+		if ( ! $manifest_data ) {
+			\WP_CLI::error( 'Manifest content is not valid json' );
+		}
+
+		return $import->adapt_manifest_structure( $manifest_data );
```
Comment (davseve) [inline core/app/modules/import-export/wp-cli.php:250]: Consider to re move the `import` instance

File: core/app/modules/import-export/wp-cli.php [hunk 2]
```diff
@@ -216,4 +225,62 @@ private function create_temp_file_from_url( $url ) {
...
+	}
+
+	/**
+	 * Handle the import process of plugins.
+	 *
+	 * Returns a string contains the successfully imported and activated plugins.
```
Comment (davseve) [inline core/app/modules/import-export/wp-cli.php:256]: Change imported to installed

File: tests/phpunit/elementor/core/utils/test-plugins-manager.php [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+	public function test_activate__array_of_plugins_with_one_already_activated() {
+		// Arrange
+		$plugins = [ 'elementor/elementor-test', 'elementor-pro/elementor-pro-test' ];
+
+		$this->wp_api_mock
+			->expects( $this->exactly( 3 ) )
+			->method( 'is_plugin_active' )
+			->withConsecutive(
+				 [ 'elementor/elementor-test.php' ],
```
Comment (davseve) [inline tests/phpunit/elementor/core/utils/test-plugins-manager.php:157]: Fix indentation

File: tests/phpunit/elementor/core/utils/test-plugins-manager.php [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+		$plugins_manager = new Plugins_Manager();
+
+		// Act
+		$activate = $plugins_manager->activate( $plugins );
+
+		// Assert
+		$this->assertEquals( $activate['succeeded'], $plugins );
+	}
+
+	public function test_activate__single_plugin() {
```
Comment (davseve) [inline tests/phpunit/elementor/core/utils/test-plugins-manager.php:177]: Add more describable title

File: core/app/modules/import-export/wp-cli.php [hunk 5]
```diff
@@ -216,4 +225,62 @@ private function create_temp_file_from_url( $url ) {
 
 		return Plugin::$instance->uploads_manager->create_temp_file( $response['body'], 'kit.zip' );
 	}
+
+	/**
+	 * Helper to get the manifest data from the file using the 'import' instance
+	 *
+	 * @param Import $import
```
Comment (davseve) [inline core/app/modules/import-export/wp-cli.php:232]: add @since 'version number'

**Review by davseve — CHANGES_REQUESTED** [review]

File: tests/phpunit/elementor/core/utils/test-plugins-manager.php (hunk 3 above)
Comment (ManorHazaz) [reply tests/phpunit/elementor/core/utils/test-plugins-manager.php:157]: It's just here..

**Review by davseve — APPROVED** [review]

**Review by davseve — APPROVED** [review]

**Review by davseve — APPROVED** [review]

**Pull request merged**